src/pegen/python_generator.py

```diff
@@ -198,6 +198,25 @@ def visit_Forced(self, node: Forced) -> Tuple[str, str]:
             )
 
 
+Used = Set[str]
```

Comment:
I tend to find that kind of aliasing to be detrimental to the readability personally (in particular for such a simple alias). But I will let @pablogsal comment on this.